src/components/Comments/WriteComment.js

```diff
@@ -255,5 +256,10 @@ export const styles = StyleSheet.create({
   },
   emojyStyle: {
     fontSize: 18
+  },
+  switch: {
+    fontFamily: fonts.inter[400],
+    fontSize: 12,
+    color: colors.gray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new style `switch` has been added. Make sure it is used correctly in the component.
```diff
+ switch: {
+   fontFamily: fonts.inter[400],
+   fontSize: 12,
+   color: colors.gray
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->